apps/frontend/components/NavBar/NavMenuContent.tsx

```diff
@@ -0,0 +1,130 @@
+import ContentPasteIcon from '@mui/icons-material/ContentPaste';
+import StarIcon from '@mui/icons-material/Star';
+import SupervisorAccountIcon from '@mui/icons-material/SupervisorAccount';
+import {
+  Box,
+  FormControl,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+  Typography,
+} from '@mui/material';
+import Logo from 'next/image';
+import { useRouter } from 'next/router';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import { useLanguageContext } from 'context';
+import { useIsSignedInUserAdmin } from 'services/api/user/useUser';
+import { env } from 'services/env';
+
+export const NavMenuContent = (): JSX.Element => {
+  const intl = useIntl();
+  const router = useRouter();
+  const isAdmin = useIsSignedInUserAdmin();
+
+  const handleAdminClick = () => {
+    const baseURL = env('NEXT_PUBLIC_API_BASE_URL');
+    const path = new URL('/admin/login', baseURL);
+    void router.push(path);
+  };
+
+  const { language, setLanguage } = useLanguageContext();
+  const handleLanguageChange = (e: SelectChangeEvent) => {
+    const newLanguage = e.target.value;
+    setLanguage(newLanguage);
+  };
+
+  const handleRoutingClick = () => {
+    const baseURL = env('NEXT_PUBLIC_BASE_PATH');
```

Comment:
https://nextjs.org/docs/api-reference/next.config.js/redirects

Base url is needed only for external redirect